pkg/operator/ceph/cluster/controller.go

```diff
@@ -195,8 +195,12 @@ func (c *ClusterController) onK8sNodeAdd(obj interface{}) {
 	}
 
 	for _, cluster := range c.clusterMap {
+		if k8sutil.NodeIsTolerable(*newNode, cephv1.GetOSDPlacement(cluster.Spec.Placement).Tolerations, true) == false {
```

Comment:
On node addition, well-known taints should not be ignored. If a node is added and facing memory pressure, for instance, Rook should *not* begin placing OSDs on the node.